Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -405,11 +405,11 @@ static void getImageBytesFromVideoFrame(WebGPU::Queue& backing, const RefPtr<Vid
     uint8_t permuteMap[4] = { 2, 1, 0, 3 };
     vImagePermuteChannels_ARGB8888(&bgra, &bgra, permuteMap, kvImageNoFlags);
 
-    return callback(byteSpan.first(sizeInBytes), width, height);
+    callback(byteSpan.first(sizeInBytes), width, height);
```

Comment:
This line seems unnecessarily changed unless I am misunderstanding, but also it is fine without the return

Reply:
Yeah this was drive-by cleanup as the method is declared as void.

Reply:
it was like that intentionally for consistency with the other return statements but I am also fine with the change :)

Reply:
I see, I can undo it before landing. Though note that `getImageBytesFromImageBuffer` doesn't have a return statement either. But maybe that's the inconsistency from your perspective?

Reply:
Oh yes it doesn't really matter so much.

The thing it prevents is someone changes the return type of `ImageDataCallback` but then fails to change the return type of the calling function (`getImageBytesFromVideoFrame`) which the `return someFunctionReturningVoid` style prevents.

In this case it doesn't matter due to the earlier above return statements which would catch the same issue. And certainly there are other ways to prevent these mistakes too (`WARN_UNUSED_RETURN`). It would be nice if `WARN_UNUSED_RETURN` was the default and then no need to have this pattern which is arguably confusing.